README.md

```diff
@@ -1,12 +1,30 @@
-# plakar
+# plakar - Effortless backup
 
-## Warning
+## Introduction
 
-Plakar is currently **NOT** production ready yet but it is most definitely stable enough to be tested by others.
+plakar is designed for effortless and secure data protection. It provides an intuitive, powerful, and scalable backup solution.
 
-Feel free to give it a try, give feedback on what you like/dislike and report bugs.
+plakar is:
+- **effortless**: easy to use
+- **secure**: provide true end-to-end encryption for data and metadata
+- **reliable**: backups are stored in an immutable repository
+- **scalable vertically**: back up and restore very large data set with little RAM
+- **scalable horizontally**: high concurrency support, multiple backups in a single repository
+- **searchable**: browse your snapshots from plakar UI, sort, search and compare data
+- **fast**: backup, check, sync and restore are fast (beta is optimized for large data sets, optimization for small one are in progress)
+- **efficient**: more restore points, less storage space, with unmatched deduplication and compression rates
+- **Open Source and well maintained**: launched in 2021, open source forever and now maintained by Plakar Korp
 
-Be aware that at this point you may still face some bugs and you may need to rebuild your backups from scratch every now and then as we bring improvements to the storage format.
+plakar is providing useful features:
+- **instant recovery**: mount instantly large backup on any devices
+- **distributed backup**: synchronize your backup repository to implement simple (3,2,1 rule) or complex (push, pull, sync) strategies
+- **granular restore**: restore a complete snapshot or only one file
+- **cross-storage restore**: back up from S3 to restore on a file system
```

Comment:
- **cross-storage backup**: back up any source of data: filesystem, S3, SFTP, and more to come
- **cross-storage restore**: restore a backup to any destination: filesystem, S3, SFTP, and more to come